src/picosvg/svg_reuse.py

```diff
@@ -272,7 +272,7 @@ def _round(affine, s1, s2):
     rotate_s2vec1_off_x = Affine2D.identity().rotate(s2_vec1_angle)
 
     affine = Affine2D.compose_ltr(
-        (s1_to_origin, s1_vec1_to_s2_vec2, rotate_s2vec1_onto_x)
+        (s1_to_origin,  s1_vec1_to_s2_vec1, rotate_s2vec1_onto_x)
```

Comment:
remember to `black` it out when you're finished editing

Reply:
Ran black